examples/chefapi_examples/recipes/chefapi.rb

```diff
@@ -4,33 +4,3 @@
 package 'git'
 
 package 'golang'
-
-directory '/go/src/github.com/go-chef' do
```

Comment:
Switching to using synced folders meant we did not need to fetch any of these resources from git.

However, I am not sure how the dependencies for `go-chef` are fetched. The tests pass but there is never a `go get` ran anywhere to fetch the dependencies.